src/main/java/org/broadinstitute/hellbender/utils/report/GATKReportTable.java

```diff
@@ -151,17 +151,6 @@ public GATKReportTable(BufferedReader reader, GATKReportVersion version) {
     }
 
     /**
-     * Construct a new GATK report table with the specified name and description
-     *
-     * @param tableName        the name of the table
-     * @param tableDescription the description of the table
-     * @param numColumns       the number of columns in this table
-     */
-    public GATKReportTable(final String tableName, final String tableDescription, final int numColumns) {
-        this(tableName, tableDescription, numColumns, TableSortingWay.SORT_BY_ROW);
-    }
-
-    /**
      * Construct a new GATK report table with the specified name and description and whether to sort rows by the row ID.
```

Comment:
Recommend restoring this constructor so that `GATKReportTable` has a default sorting method.